ci_support/run_docker_build.sh

```diff
@@ -70,4 +70,11 @@ yum install -y devtoolset-2-gcc-gfortran
     set +x
     conda build /recipe_root --quiet || exit 1
     upload_or_check_non_existence /recipe_root conda-forge --channel=main || exit 1
+
+    set -x
+    export CONDA_NPY=110
```

Comment:
Are we really doing numpy 1.10 for Python 3.6? Numpy itself doesn't support that (although it may work with maybe some test failures). The first wheels for Python 3.6 are when numpy 1.12.0 comes out I think.